tests_integration/lib/integration_test_helper.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import shutil
+import subprocess
+import tempfile
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from github import Github
+
+from conda_forge_tick.git_utils import GitCli
+from conda_forge_tick.utils import (
+    run_command_hiding_token,
+)
+from tests_integration.lib.shared import (
+    FEEDSTOCK_SUFFIX,
+    GitHubAccount,
+    get_github_token,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+class IntegrationTestHelper:
```

Comment:
Is there a real need for a class here? I don't see much shared state and it could just be a collection of public functions (plus their private counterparts). Or is it more about typing?

Reply:
We could convert everything into public functions, but this approach makes it very clear IMO that you are supposed to use the instance methods of the helper within a test case. Also, we keep the flexibility of adding some state that we might need in the future (e.g., the ID of a pull request that has been created).